ArduSub/compassmot.cpp

```diff
@@ -52,8 +52,7 @@ uint8_t Sub::mavlink_compassmot(mavlink_channel_t chan)
     }
 
     // check if radio is calibrated
-    pre_arm_rc_checks();
-    if (!ap.pre_arm_rc_check) {
+    if (!arming.rc_check()) {
```

Comment:
Don't you want to use `rc_check(false)` here? Otherwise you'll get two MAVLink messages saying mostly the same.